src/ml_gym/starter.py

```diff
@@ -42,8 +42,9 @@ def _create_gym(process_count: int, device_ids, log_std_to_file: bool) -> Gym:
 
     @staticmethod
     def _setup_logging_environment(log_dir_path: str):
-        queue = Queue()
-        QueuedLogging.start_logging(queue, log_dir_path)
+        if QueuedLogging._instance is None:
```

Comment:
we already have this check within QueuedLogging, is there a reason why you added this here again?

Reply:
because if it is none, it will raise the error SingletonAlreadyInstantiatedError
    `def start_logging(cls, log_msg_queue: Queue, log_dir_path: str):
        if cls._instance is None:
            cls._instance = QueuedLogging(log_msg_queue, log_dir_path)
            cls._instance._start_listening()
        else:
            raise SingletonAlreadyInstantiatedError()`